openverse_catalog/dags/common/extensions.py

```diff
@@ -0,0 +1,18 @@
+from typing import Literal
+
+
+EXTENSIONS = {
+    "image": ["jpg", "jpeg", "png", "gif", "bmp", "webp", "tiff", "tif", "svg", "webp"],
+    "audio": ["mp3", "ogg", "wav", "aiff", "flac", "aac", "m4a", "wma", "mp4", "m4b"],
+}
+
+
+def extract_filetype(url: str, media_type: Literal["image", "audio"]) -> str | None:
```

Comment:
We had a [lengthy discussion](https://github.com/WordPress/openverse-catalog/pull/496#discussion_r868573848) about this `Literal` in a different PR 😅 I think we should probably just throw it in `constants.py` and use it from there! Even if we have to repeat ourselves, better than defining this everywhere we need it

Reply:
FWIW I think I'm convinced 😄 At any rate we can always iterate on that if we come up with a cleaner way to define the type. Better repeating ourselves once than in many places.

Reply:
I remember seeing the start of that discussion and have spent some time trying to find it in the code, so thank you for the link!
I'll just remove the type hints from these PRs, and we can implement it in other PRs :)

Reply:
I made an issue for it! https://github.com/WordPress/openverse-catalog/issues/563